### PR TITLE
Create DHCPAudit Parser Function

### PR DESCRIPTION
**Pull request by mayurrr98**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Parsers/DHCPAuditParserFunction [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// This parser takes raw DHCP Audit logs and parses the data into a normalized schema
+
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as DHCP_Audit_CL_ActivityReport.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. DHCP_Audit_CL_ActivityReport | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let DHCP_Audit_CL_view = view () {
+DHCP_Audit_CL
+| extend splitted=parse_csv(RawData)
+| mv-expand EventOriginalUid=splitted[0], Date=splitted[1], Time=splitted[2], EventOriginalType=splitted[3] ,SrcIpAddr=splitted[4], SrcHostname=splitted[5], SrcMacAddr=splitted[6], SrcUsername=splitted[7], DhcpSessionId=splitted[8], EventResultDetails=splitted[9], DhcpSrcDHCId=splitted[12], DhcpVendorClass=splitted[13], DhcpVendorClassId=splitted[14], DhcpUserClass=splitted[15], DhcpUserClassId=splitted[16]
```
Comment (oshezaf) [inline Parsers/DHCPAuditParserFunction:17]: Not sure why use `mv-expand` instead of just `extend`.

File: Parsers/DHCPAuditParserFunction [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Title:           DHCP Audit Data Parser
+// Author:          Microsoft
```
Comment (oshezaf) [inline Parsers/DHCPAuditParserFunction:2]: Should be the author of the parser, not the source.

**Review by oshezaf — CHANGES_REQUESTED** [review]
Thanks for the contribution. 

Apart from the comments on specific lines, I would appreciate also the following: 

- Please add sample data to the `Sample Data/Custom` folder, so I can verify the parser.
- Please add to the description how to set up collection.
- Please rename: put in a folder under parsers and add an extension, for example `parsers/DHCP_OMS/DHCPAudit.txt`

**Comment by v-jayakal** [comment]
@mayurrr98  - Can you please the comments provided, Thanks

**Comment by NikTripathi** [comment]
@mayurrr98 Can you please resolve the comments.

**Comment by v-sabiraj** [comment]
Hey @mayurrr98 , will you please resolve the requested changes, thanks.

**Comment by v-sabiraj** [comment]
Hey @mayurrr98 , will you please resolve the requested changes, thanks.

**Comment by v-sabiraj** [comment]
As @ashishsyal, has discussed with author @mayurrr98 we are closing this PR.

**Pull request closed without merging**